packages/core/src/schema/models/providerLogs.ts

```diff
@@ -31,10 +33,10 @@ export const providerLogs = latitudeSchema.table('provider_logs', {
       onUpdate: 'cascade',
     }),
   model: varchar('model'),
-  config: json('config').notNull(),
-  messages: json('messages').notNull(),
-  responseText: text('response_text'),
-  toolCalls: json('tool_calls'),
+  config: json('config').$type<PartialConfig>().notNull(),
+  messages: json('messages').$type<Message[]>().notNull(),
+  responseText: text('response_text').$type<string>().default(''),
+  toolCalls: json('tool_calls').$type<ToolCall[]>().default([]),
```

Comment:
This is awesome. Drizzle can type `json` columns

Reply:
oh wow

Reply:
I added defaults in `responseText` and `toolCalls` I think it doesn't hurt